src/gt4py/next/program_processors/runners/dace/gtir_dataflow.py

```diff
@@ -613,35 +614,56 @@ def _visit_if_branch_arg(
             if_branch_state: The state inside the nested SDFG where the if branch is lowered.
             param_name: The parameter name of the input argument.
             arg: The input argument expression.
+            deref_on_input_memlet: When True, the given iterator argument can be dereferenced on the input memlet.
```

Comment:
```suggestion
            deref_on_input_memlet: When True, the given iterator argument can be dereferenced on the input memlet. This means that the values are copied into a temporary storage which is passed into the nested SDFG.
```
I think the description of this option should have a bit more information.

Reply:
The change you propose is not correct. We are not allocating temporary storage, we are just narrowing the memlet subset.